pipeline/steps/tf-install.yaml

```diff
@@ -1,7 +1,7 @@
 parameters:
   - name: version
     type: string
-    default: '0.12.24'
+    default: '0.15.4'
```

Comment:
should update to 1.x, (separate PR though)

Reply:
Added ticket to resolve this. thank you